mlkem/polyvec.c

```diff
@@ -237,6 +239,9 @@ void polyvec_basemul_acc_montgomery_cached(poly *r, const polyvec *a, const poly
     }
 
     poly_reduce(r);
+    #else /* MLKEM_USE_AARCH64_ASM */
+    polyvec_basemul_acc_montgomery_cached_asm((int16_t *) r, (int16_t *) a, (int16_t *) b, (int16_t *) b_cache);
```

Comment:
Do we prefer `(int16_t *) r` or `r->coeffs`? Let's do one of the two consistently. I have a slight preference for the latter. It's unlikely that the `poly` struct will change, but who knows.

Reply:
Ok